spec/models/question/invalid_subject_spec.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe Question::InvalidSubject do
```

Comment:
ah i see, you added the invalid spec here 🤔

Reply:
Invalid specs are several places, I believe... I'll try to comment & note them. This verifies that if InvalidSubject is set, it will always respond as not valid. It doesn't address the setting of it.